packages/desktop/components/popups/Popup.svelte

```diff
@@ -189,46 +188,39 @@
 </script>
 
 <svelte:window on:keydown={onKey} />
-{#if $mobile && !fullScreen}
-    <Drawer opened zIndex="z-40" preventClose={hideClose} on:close={() => closePopup($popupState?.preventClose)}>
-        <div bind:this={popupContent} class="p-8">
-            <svelte:component this={types[type]} {...props} />
-        </div>
-    </Drawer>
-{:else}
-    <popup
-        in:fade={{ duration: transition ? 100 : 0 }}
-        class={`flex items-center justify-center fixed ${
-            $platform === PlatformOption.Windows ? 'top-9' : 'top-0'
-        } left-0 w-screen p-6 ${overflow ? '' : 'overflow-hidden'}
+
+<popup
+    in:fade={{ duration: transition ? 100 : 0 }}
+    class={`flex items-center justify-center fixed ${
+        $platform === PlatformOption.Windows ? 'top-9' : 'top-0'
+    } left-0 w-screen p-6 ${overflow ? '' : 'overflow-hidden'}
                 h-full z-20 ${
                     fullScreen
                         ? 'bg-white dark:bg-gray-900'
                         : 'bg-gray-800 bg-opacity-70 dark:bg-black dark:bg-opacity-50'
                 } ${$mobile && 'z-40'}`}
```

Comment:
you could also remove this dependency of `$mobile` here and remove the import

Reply:
Nice catch!